app/domain/authentication/usecase/login.go

```diff
@@ -0,0 +1,25 @@
+package usecase
+
+import (
+	"errors"
+	"github.com/patriciapedrosaa/transfer-me/app/domain/vos"
+)
+
+var ErrInvalidCredentials = errors.New("incorrect username or password")
+
+type LoginInputs struct {
+	CPF    string
+	Secret string
+}
+
+func (a Authentication) CheckLogin(inputs LoginInputs) (bool, error) {
+	account, err := a.accountRepository.GetByCpf(inputs.CPF)
```

Comment:
I believe it's not intended for authentication to be able to list accounts, shouldn't this be a responsibility of the account's domain only?